hyfer-frontend/src/Pages/Timeline/TimeLine.js

```diff
@@ -31,7 +31,7 @@ export default class TimeLine extends Component {
     isATeacher: false,
     tab: 'readme',
     readme: null,
-    repoName: null,
+    repoName: [],
```

Comment:
This does not seem to be a correct solution. The `repoName` is supposed to be the name of the repo (a string). So it can never be an array. Apparently, the first repo displayed (if nothing is clicked) is the `curriculum` repo. So it would make more sense to me if the `repoName` is initialised to `curriculum` rather than `[]`.